packages/eid/app/helpers/getPatientUUID.ts

```diff
@@ -4,15 +4,32 @@ let CM = config.ConnectionManager.getInstance();
 export default class GetPatient {
   async getPatientUUIDUsingIdentifier(params: string) {
     let amrsCON = await CM.getConnectionAmrs();
-    let sql = `select p.uuid from amrs.patient_identifier pi left join amrs.person p on p.person_id=pi.patient_id where pi.identifier='${params}' and pi.voided=0`;
-    let result: string = await CM.query(sql, amrsCON);
+    let sql = `select p.uuid from amrs_migration.patient_identifier pi left join amrs_migration.person p on p.person_id=pi.patient_id where pi.identifier='${params}' and pi.voided=0`;
+    let result: any = await CM.query(sql, amrsCON);
+    await CM.releaseConnections(amrsCON);
+    return result;
+  }
+  async getPatientUUIDUsingCCCNo(params: string) {
+    let amrsCON = await CM.getConnectionAmrs();
+    let sql = `select p.uuid from amrs_migration.patient_identifier pi left join amrs_migration.person p on p.person_id=pi.patient_id where pi.identifier='${params}' and pi.identifier_type=28 and pi.voided=0`;
+    let result: any = await CM.query(sql, amrsCON);
```

Comment:
Can we merge this two functions and make the query string dynamic?

Reply:
done